src/store.js

```diff
@@ -1,36 +1,39 @@
 import { defineStore } from 'pinia'
-import { getDatabaseData } from './api/getDatabaseOld'
-import storageTitle from './storage/adapters/title'
+import storageFolder from './storage/adapters/folder'
 import storageTask from './storage/adapters/task'
 
 export const useDataStore = defineStore('data', {
 	state: () => ({
-		data: [],
-		title: [],
+		folder: [],
 		task: []
 	}),
 
 	getters: {
-		getTitleById: state => id => {
-			const title = state.title.find(title => title.id === id)
-			if (!title) {
+		getFolderById: state => id => {
+			if (!state.folder && !Array.isArray(state.folder)) {
+				return []
```

Comment:
getFolderById should return null or object {...folder, tasks: tasks}. Why return []?